package.json

```diff
@@ -105,6 +105,7 @@
     "clean.screenshots": "rimraf test/end-to-end/screenshot/builds test/end-to-end/screenshot/images",
     "clean:scripts": "rimraf scripts/build",
     "lint": "eslint bin/* scripts/*.ts scripts/**/*.ts src/*.ts src/**/*.ts src/**/*.tsx test/wdio/**/*.tsx",
+    "lint.fix": "eslint bin/* scripts/*.ts scripts/**/*.ts src/*.ts src/**/*.ts src/**/*.tsx test/wdio/**/*.tsx --fix",
```

Comment:
```suggestion
```
nitpick - but I don't think we need to add this. Don't want the list to be longer that it has to be :)